devmem/__main__.py

```diff
@@ -121,11 +133,16 @@ def main() -> int:
         if args.verbose:
             print(
                 "Value after write:\t{0}".format(
-                    mem.read(0x0, args.num).hexdump(args.word_size)
+                    mem.read(0x0, args.num).hexdump(words_per_row)
                 )
             )
+
+    elif args.binary:
+        buf = mem.read(0x0, args.num)
+        sys.stdout.buffer.write(buf.data)
```

Comment:
`sys.stdout.buffer.write` do not work on Python 2.7
`sys.stdout.write` for Python 2.7